adobe/react-spectrum
# RSP-1379 Days appear outside of Calendar

1. Adjust window to be short height
2. Open Calendar from Datepicker
3. Days are outside of calendar border; should be scrollable like tray version

Popover should have overflow:auto

Navigating through dates should scroll focused date into view.


Closes [RSP-1379](https://jira.corp.adobe.com/browse/RSP-1379)

## ✅ Pull Request Checklist:

- [x] Included link to corresponding Issue [RSP-1379](https://jira.corp.adobe.com/browse/RSP-1379).
- Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- Updated documentation (if it already exist for this component).

## 📝 Test Instructions:
1. Open http://localhost:9003/?path=/story/datepicker--default
2. Adjust window to be short height
3. Open Calendar from Datepicker
4. Popover should be scrollable like tray version
5. Use arrow keys to navigate by date. 
6. Focused date should scroll into view within Popover.

## 🧢 Your Team:

Accessibility
